bibliopixel/drivers/hue.py

```diff
@@ -62,7 +61,7 @@ def _rgb2hs(self, rgb):
     def _send_packet(self):
         for i in range(len(self._ids)):
             h, s = self._rgb2hs(self._colors[i + self._pos])
-            bri = self._brightness
+            bri = min(254, self._brightness)
```

Comment:
Oops... didn't pay attention to that being the hue driver not some other internal hue calculations. Nevermind, carry on.